dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/synch/DefaultSynchronizationManager.java

```diff
@@ -78,7 +78,8 @@
  * @author Lars Helge Overland
  */
 public class DefaultSynchronizationManager
-    implements SynchronizationManager
+    implements
+    SynchronizationManager
```

Comment:
Not sure if we have specific conventions for it, but I think implements and SyncManager should be on same line